### ISSUE TITLE
`odo component list` should error out

### ISSUE THREAD

**Issue by surajnarwade** [redhat-developer/odo#998]
[kind/bug]

## Expected behavior

```
 $ odo component list
Please use `odo list`
```

## Actual behavior

```
$ odo component list
Component list does not exist in application nodejs-ex-znfa
```

**Comment by kadel** [comment]
this is tricky, as `odo component <name>` is there to switch active component and someone could create a component that is named `list`

**Comment by kadel** [comment]
this is related to #953  and #779

**Comment by kadel** [comment]
Closing this as it will be solved as #779
